var/spack/repos/builtin/packages/glusterfs/package.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Glusterfs(AutotoolsPackage):
+    """Gluster is a software defined distributed storage that can scale to
+    several petabytes. It provides interfaces for object, block and file
+    storage."""
+
+    homepage = "https://www.gluster.org/"
+    url      = "https://download.gluster.org/pub/gluster/glusterfs/7/7.3/glusterfs-7.3.tar.gz"
+    list_depth = 1
```

Comment:
I think this should be:
```python
list_url = "https://download.gluster.org/pub/gluster/glusterfs/"
list_depth = 2
```
You can test this with `spack versions glusterfs`.

Reply:
Fix it in: https://github.com/spack/spack/pull/15565/commits/020ddc51e659b7a3d46b92e40b6a49fc863cdc11